src/HCSR04.h

```diff
@@ -31,7 +33,8 @@ class UltraSonicDistanceSensor {
      */
     double measureDistanceCm(float temperature);
```

Comment:
I would mention here that it takes into consideration maxDistanceCm and maxTimeoutMicroSec and takes both into account.